IBAnimatable/PresentationAnimationType.swift

```diff
@@ -8,17 +8,17 @@ import UIKit
 /**
  Predefined Transition Animation Type
  */
-public enum PresentationAnimationType {
+public enum PresentationAnimationType: IBEnum {
   case flip
   case crossDissolve
   case cover(fromDirection: TransitionAnimationType.Direction)
```

Comment:
We may be able to use `case cover(from: TransitionAnimationType.Direction)` based on **Choose parameter names to serve documentation** from **API Design Guidelines**, because the type can be found in `TransitionAnimationType.Direction`